webdav/src/main/java/ch/cyberduck/core/dav/DAVSession.java

```diff
@@ -348,24 +372,34 @@ public <T> T _getFeature(final Class<T> type) {
         return super._getFeature(type);
     }
 
-    private final class MicrosoftIISFeaturesResponseHandler extends ValidatingResponseHandler<Void> {
+    public static final class HttpCapabilities {
+        /**
+         * Support for Expect: Continue
+         */
+        public boolean expectcontinue;
+        public boolean iis;
+
+        public HttpCapabilities(final PreferencesReader preferences) {
+            this.expectcontinue = preferences.getBoolean("webdav.expect-continue");
+        }
+
+        public HttpCapabilities withExpectcontinue(final boolean expectcontinue) {
+            this.expectcontinue = expectcontinue;
+            return this;
+        }
+
+        public HttpCapabilities withIis(final boolean iis) {
```

Comment:
Rename to `withIIS` to make it consistent with the classes such as `MicrosoftIISPersistentAuthResponseInterceptor`?